src/heads-up/components/app-tooltip.vue

```diff
@@ -15,234 +15,214 @@
-      margin-bottom: 0;
-    }
-
-  .tooltip[x-placement^="bottom"] {
-    margin-top: 5px;
-  }
+.tooltip[x-placement^="top"] .tooltip-arrow {
+  bottom: -5px;
+  margin: 0 5px;
+}
 
-  .tooltip[x-placement^="bottom"] .tooltip-arrow {
-      border-width: 0 5px 5px 5px;
-      border-left-color: transparent !important;
-      border-right-color: transparent !important;
-      border-top-color: transparent !important;
-      top: -5px;
-      left: calc(50% - 5px);
-      margin-top: 0;
-      margin-bottom: 0;
-    }
-
-  .tooltip[x-placement^="right"] {
-    margin-left: 5px;
-  }
+.tooltip[x-placement^="bottom"] {
+  margin-top: 5px;
+}
 
-  .tooltip[x-placement^="right"] .tooltip-arrow {
-      border-width: 5px 5px 5px 0;
-      border-left-color: transparent !important;
-      border-top-color: transparent !important;
-      border-bottom-color: transparent !important;
-      left: -5px;
-      top: calc(50% - 5px);
-      margin-left: 0;
-      margin-right: 0;
-    }
-
-  .tooltip[x-placement^="left"] {
-    margin-right: 5px;
-  }
+.tooltip[x-placement^="bottom"] .tooltip-arrow {
+  top: -5px;
+  margin: 0 5px;
+}
 
-  .tooltip[x-placement^="left"] .tooltip-arrow {
-      border-width: 5px 0 5px 5px;
-      border-top-color: transparent !important;
-      border-right-color: transparent !important;
-      border-bottom-color: transparent !important;
-      right: -5px;
-      top: calc(50% - 5px);
-      margin-left: 0;
-      margin-right: 0;
-    }
-
-  .tooltip[aria-hidden='true'] {
-    visibility: hidden;
-    opacity: 0;
-    transition: opacity .15s, visibility .15s;
-  }
+.tooltip[x-placement^="right"] {
+  margin-left: 5px;
+}
 
-  .tooltip[aria-hidden='false'] {
-    visibility: visible;
-    opacity: 1;
-    transition: opacity .15s;
-  }
+.tooltip[x-placement^="right"] .tooltip-arrow {
+  left: -5px;
+  margin: 5px 0;
+}
 
-  .vue-tooltip {
-    width: 150px;
-    padding: 5px 8px;
-    background-color: var(--selection-fg-color);
-    border-radius: 2px;
-    color: hsl(0, 0%, 20%);
-    line-height: 14px;
-    text-align: left;
-    filter: drop-shadow(0 1px 2px hsla(0, 0%, 0%, 0.3));
-    border: 1px solid hsla(0, 0%, 0%, 0.1);
-  }
+.tooltip[x-placement^="left"] {
+  margin-right: 5px;
+}
 
-  .vue-tooltip::after {
-    content: "";
-    position: absolute;
-    top: 100%;
-    left: 0;
-    width: 100%;
-    height: 1.2rem;
-  }
+.tooltip[x-placement^="left"] .tooltip-arrow {
+  right: -5px;
+  margin: 5px 0;
+}
 
+.tooltip[aria-hidden="true"] {
+  visibility: hidden;
+  opacity: 0;
+  transition: opacity 0.15s, visibility 0.15s;
+}
 
-  .-theme-with-dark-background .vue-tooltip {
-    background-color: var(--toolbar-bg-color);
-    color: var(--base-color);
-    border: 1px solid var(--base-color);
-  }
+.tooltip[aria-hidden="false"] {
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.15s;
+  box-shadow: 0px 0px 10px -4px rgba(0, 0, 0, 1),
+    0px 0px 6px -4px rgba(0, 0, 0, 1);
+}
+/* 
```

Comment:
All commented out code can be removed, if it is needed back again, we can look it up in this commit